source/framework/vx_node.c

```diff
@@ -223,15 +223,40 @@ static vx_status ownRemoveNodeInt(const vx_node *n)
     return status;
 }
 
-static void ownNodeUserKernelSetParamsAccesible(vx_reference params[], vx_uint32 num_params, vx_bool is_accessible)
+static void ownNodeUserKernelSetParamsAccesible(vx_kernel kernel, vx_reference params[], vx_uint32 num_params, vx_bool is_accessible)
```

Comment:
Can we also change the boolean from "vx_bool is_accessible" to "vx_bool kernel_entry" ... this will make it more generic about changing flags related to entering the kernel vs exiting the kernel.

Reply:
Gentle ping on this

Reply:
@raphacan Ping on this review comment.

Reply:
done